academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md

```diff
@@ -2,24 +2,30 @@
 title: MAKES troubleshooting guide
 description: Addresses common problems found with MAKES
 ms.topic: reference
-ms.date: 2020-02-10
+ms.date: 2020-04-15
 ---
 
 # Troubleshoot Common MAKES command line tool issues
 
 ## Azure login failure due to multiple subscriptions or multiple tenants being tied to a single Azure account
 
-If you have multiple azure tenants associated with a single azure account, you'll need to specify the Azure Active Directory directory domain name associated with the Azure Subscription that you would like to use to execute the command. (e.g. "--AzureActiveDirectoryDomainName constco.onmicrosoft.com").  
+If you have multiple Azure tenants associated with a single Azure account, you'll need to specify the Azure Active Directory directory domain name associated with the Azure Subscription that you would like to use to execute the command. (e.g. "--AzureActiveDirectoryDomainName constco.onmicrosoft.com").  
 
-You can find this information by logging into azure portal, searching for "Azure Active Directory" and viewing resource detail page. The domain name to use will be in the "Overview" section at the top left.
+You can find this information by logging into Azure Portal, searching for "Azure Active Directory" and viewing resource detail page. The domain name to use will be in the "Overview" section at the top left.
```

Comment:
"You can find this information by logging into **the** Azure Portal, searching for 'Azure Active Directory' and viewing **the** resource detail page."